hypervisor/arch/x86/guest/vlapic.c

```diff
@@ -202,6 +202,30 @@ static inline void vlapic_build_x2apic_id(struct acrn_vlapic *vlapic)
 	lapic->ldr.v = (cluster_id << 16U) | (1U << logical_id);
 }
 
+static inline uint32_t vlapic_find_isrv(const struct acrn_vlapic *vlapic)
+{
+	const struct lapic_regs *lapic = &(vlapic->apic_page);
+	uint32_t i, val, bitpos, isrv = 0U;
+	const struct lapic_reg *isrptr;
+
+	if (vlapic->isr_nr != 0U) {
+		isrptr = &lapic->isr[0];
+
+		/* i ranges effectively from 7 to 1 */
+		for (i = 8U; i > 1U; ) {
+			i--;
+			val = atomic_load32(&isrptr[i].v);
```

Comment:
use lapic->isr directly here?

Reply:
Why we need atomic_load32? How about if we directly read it ?

Reply:
The current coding style is using this.  Will change it.